src/components/ChangelogDialog/components/Picture/Picture.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {block} from '../../../utils/cn';
+import {Loader} from '../../../Loader';
+
+import './Picture.scss';
+
+const b = block('changelog-dialog-picture');
+
+type LoadingState = 'loading' | 'loaded' | 'error';
+
+export interface PictureProps {
+    className?: string;
+    src: string;
+    ratio: number;
+}
+
+export function Picture({className, src, ratio}: PictureProps) {
+    const [loadingState, setLoadingState] = React.useState<LoadingState>('loading');
+
+    React.useEffect(() => {
+        // in SSR case
+        if (typeof window === 'object') {
+            setLoadingState('loading');
+
+            const img = new Image();
+            img.onload = () => {
+                setLoadingState('loaded');
+            };
+            img.onerror = img.onabort = () => {
+                setLoadingState('error');
+            };
+            img.src = src;
+        }
+    }, [src]);
+
+    if (loadingState === 'error') {
+        return null;
+    }
+
+    return (
+        <div className={b(null, className)} style={{paddingBottom: `${ratio * 100}%`}}>
+            {loadingState === 'loading' ? (
+                <div className={b('loader')}>
+                    <Loader size="s" />
+                </div>
+            ) : null}
+            <div
+                className={b('image', {visible: loadingState === 'loaded'})}
```

Comment:
is it always visible here?

Reply:
It is visible when image is loaded. Until this moment user sees placeholder with same size as image.